glue/viewers/profile/viewer.py

```diff
@@ -1,18 +1,42 @@
 from glue.core.subset import roi_to_subset_state
+from glue.core.coordinates import LegacyCoordinates
+
+from astropy.wcs import WCS
+from astropy.visualization.wcsaxes.frame import RectangularFrame1D
 
 __all__ = ['MatplotlibProfileMixin']
 
 
+def get_identity_wcs(naxis):
+
+    wcs = WCS(naxis=naxis)
+    wcs.wcs.ctype = ['X'] * naxis
+    wcs.wcs.crval = [0.] * naxis
+    wcs.wcs.crpix = [1.] * naxis
+    wcs.wcs.cdelt = [1.] * naxis
+
+    return wcs
```

Comment:
Avoid duplication with image viewer by moving this to glue.utils? (you can make a new glue.utils.wcs module)

Reply:
Sure